db/schema.rb

```diff
@@ -345,7 +345,7 @@
     t.string "email", default: "", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.text "ecf_id"
+    t.uuid "ecf_id"
```

Comment:
will this affect any indices? will it work ok with `nil` values? also can we test this on the migration env by deploying this just invase?

Reply:
also a few failing specs to consider :)

Reply:
The indices should continue to work and it works with `nil` values. I've tested it locally.
Will run on migration env before prod merge.